### DISCUSSION TITLE
Is there a tutorial for fune-tuning contextnet model?

### DISCUSSION THREAD

**Discussion by bfss** [NVIDIA-NeMo/NeMo#4258] [General]
I would like to fine tune the contextnet model, and I have no idea for what should I prepare, a config file or other? Is there any tutorial?
Thanks~

**Comment by titu1994** [comment]
The finetuning ASR CTC model has all of the information for finetuning any RNNT model too. You can also refer to the ASR with Transducers tutorial to see how similar it is to CTC training.

**Reply by titu1994** [reply]
https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/ASR_CTC_Language_Finetuning.ipynb - CTC/RNNT finetuning

https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/ASR_with_Transducers.ipynb - Transducer training

**Reply by bfss** [reply]
> The finetuning ASR CTC model has all of the information for finetuning any RNNT model too. You can also refer to the ASR with Transducers tutorial to see how similar it is to CTC training.

Thanks a lot ~
